client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClient.java

```diff
@@ -83,6 +83,12 @@ public int register(String subject, ParsedSchema schema, int version, int id) th
   public ParsedSchema getSchemaBySubjectAndId(String subject, int id)
       throws IOException, RestClientException;
 
+  public List<ParsedSchema> getSchemas(
```

Comment:
Please change this to `default` and have the implementation `throw new UnsupportedOperationException`.   We have some other implementations of `SchemaRegistryClient` outside of this project that break otherwise.

Reply:
Just pushed a new commit with this change.  Thank you for having a look!